domain/src/commonMain/kotlin/com/islamversity/domain/repo/SoraSearchRepo.kt

```diff
@@ -0,0 +1,28 @@
+package com.islamversity.domain.repo
```

Comment:
Is it ok that a specific repository class like `SoraSearchRepo ` has been residing in the domain module, isn't better to be in its own module?